blocks/results-list-block/index.story.jsx

```diff
@@ -26,6 +26,8 @@ const sampleData = {
 	showHeadline: false,
 	showImage: false,
 	showItemOverline: false,
+	overline: "overline",
```

Comment:
This is causing storybook to render all items with the same overline - the overline is driven by the story data